migration/src/main/java/com/intershop/customization/migration/MigrateConfigResurces.java

```diff
@@ -0,0 +1,226 @@
+                            {
+                                targetTpe = "user";
+                            }
+                            if (!"".equals(targetTpe))
+                            {
+                                // convert resource file to properties file
+                                String targetName = targetFile.toFile().getAbsolutePath();
+                                targetName= targetName.replace(".resource", ".properties");
+                                Path target = Paths.get(targetName);
+                                convertResourceFile(targetTpe, source, target);
+                                Files.delete(source);
+                                LOGGER.debug("Convered file {} ==>  {}.", source, target);
+                            }
+                            else
+                            {
+                                // other than resouirce files are just moved to their new location
+                                LOGGER.warn("file {} not yet handled.", source );
+                            }
+                        }
+                        else
+                        {
+                            LOGGER.debug("to be deleted {} ==>  {}.", source);
+                            toRemove.add(source);
+                        }
+                    }
+                    catch(Exception e)
+                    {
+                        throw new RuntimeException(e);
+                    }
+                });
+           }
+
+
+           toRemove.stream().filter(this::isEmpty).forEach(this::delete);
+
+        }
+        catch(IOException e)
+        {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private Set<String> getArtifactTypes()
+    {
+        return Set.of("config",  "sites");
+    }
+
+    private Path getTarget(Path cartridgeName, Path source, Path sourceMain)
+    {
+        String filName = source.getName(0).toString();
+        Path  targetPath =sourceMain.resolve("resources/resources").resolve(cartridgeName);
+        if(targetPath.toString().contains("resources"+java.io.File.separator+"resources"+java.io.File.separator+cartridgeName))
+        {
+            // targetPath = targetPath.resolve("resources/resources").resolve(cartridgeName);
+            switch(filName)
+            {
+                case "domains":
+                    Path targetSubDomains = source.subpath(2, source.getNameCount());
+                    targetPath = targetPath.resolve(targetSubDomains);
+                    break;
+                case "system":
+                    Path targetSubConfig = source.subpath(1, source.getNameCount());
+                    targetPath = targetPath.resolve(targetSubConfig);
+                    break;
+                case "cartridge":
+                    Path targetSubSites = source.subpath(2, source.getNameCount());
+                    targetPath = targetPath.resolve(targetSubSites);
+                    break;
```

Comment:
`domains` and `cartridge` are equal. so they can be merged:
```suggestion
                case "domains", "cartridge":
                    Path targetSubDomains = source.subpath(2, source.getNameCount());
                    targetPath = targetPath.resolve(targetSubDomains);
                    break;
                case "system":
                    Path targetSubConfig = source.subpath(1, source.getNameCount());
                    targetPath = targetPath.resolve(targetSubConfig);
                    break;
```

Reply:
code optimization will be done in refactoring. set to resolved for now